firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -269,6 +274,29 @@ static percent_t currentEtbDuty;
 
 		etb1.dcMotor.Set(ETB_PERCENT_TO_DUTY(currentEtbDuty));
 
+	   	float etbError = absF(targetPosition - actualThrottlePosition); // Used for I-Term limiter and Error-detection
+
+
+        // ETB Deviation monitoring
+        if(etbError < 3 ){
+          start_time = chVTGetSystemTimeX(); //Updates timestamp as long as ETB-Error stays within 3% deviation
```

Comment:
please don't directly use ChibiOS's time API - we have our own

Reply:
currentTimeMillis() ?

and efitimems_t instead f systime_t?